docs/DEVELOPING.md

```diff
@@ -343,6 +343,139 @@ $ make ftest NAME=azure_blob_storage
 
 ℹ️ The e2e test uses default values defined in [configure Azure Blob Storage connector](#configure-azure-blob-storage-connector)
 
+
+## Oracle Database Connector
+
+The [Elastic Oracle Database connector](https://github.com/elastic/connectors-python/blob/8.7/connectors/sources/oracle.py) is provided in the Elastic connectors python framework and can be used via [build a connector](https://www.elastic.co/guide/en/enterprise-search/current/build-connector.html).
+
+### Availability and prerequisites
```

Comment:
For users who are fetching from non-system tables, can we also document the minimum privileges required for the database user for the connector to fetch data from these user tables?

Reply:
We should also include the versions and editions of Oracle database we intend to support.

Reply:
**++**

Reply:
Added note for privileges. Also, I added versions of Oracle.